### PR TITLE
T15932

### PR DESCRIPTION
**Pull request by katelk28**
Design work done on Card.DefaultFamily and Card.List (and some done on the _base.scss) file

https://phabricator.endlessm.com/T15932

### CODE REVIEW

File: data/css/modules/card/_list.scss [hunk 1]
```diff
@@ -36,27 +34,42 @@
...
 
-
     &.width-e &__title {
         font-size: 18px;
     }
 
-    &.width-f &__title {
-        font-size: 23px;
+    &.width-f {
+        &__title {
```
Comment (tchx84) [inline data/css/modules/card/_list.scss]:
```
**&.width-f &__title **
```

and 

```
&.width-f {
         &__title {
```

Does not generate the same thing, and its actually generating something wrong.

Comment (katelk28) [inline data/css/modules/card/_list.scss]: You're right, totally missed this, yikes! Fixed.

Also, added a fix to `Card.DefaultFamily` styling that I noticed was creating a weird thing in the Travel app and Prensa libre app: when put inside of `Arrangement.SquareGuys`, it wasn't showing the synopsis, so chnaged the title line amount to 1. And edited the gradient a bit.
this should be good to go!

**Comment by tchx84** [comment]
Noticed this when testing on travel app and going to a set page:

```
(com.endlessm.travel.en:24114): Gjs-WARNING **: JS ERROR: Error: No property 'source_attribute' in property list (or its value was undefined)
DefaultFamily<._main_layout/<@resource:///com/endlessm/knowledge/js/app/modules/card/defaultFamily.js:385:17
DefaultFamily<._main_layout@resource:///com/endlessm/knowledge/js/app/modules/card/defaultFamily.js:384:9
wrapper@resource:///org/gnome/gjs/modules/lang.js:178:13
DefaultFamily<.vfunc_size_allocate@resource:///com/endlessm/knowledge/js/app/modules/card/defaultFamily.js:438:1
wrapper@resource:///org/gnome/gjs/modules/lang.js:178:13
place_card@resource:///com/endlessm/knowledge/js/app/interfaces/arrangement.js:332:5
_HalfLayout<.vfunc_size_allocate/<@resource:///com/endlessm/knowledge/js/app/modules/arrangement/half.js:81:13
_HalfLayout<.vfunc_size_allocate@resource:///com/endlessm/knowledge/js/app/modules/arrangement/half.js:80:9
wrapper@resource:///org/gnome/gjs/modules/lang.js:178:13
ResponsiveMargins<.vfunc_size_allocate@resource:///com/endlessm/knowledge/js/app/modules/layout/responsiveMargins.js:132:9
wrapper@resource:///org/gnome/gjs/modules/lang.js:178:13
_parent@resource:///org/gnome/gjs/modules/lang.js:131:5
Simple<.vfunc_size_allocate@resource:///com/endlessm/knowledge/js/app/modules/window/simple.js:167:9
wrapper@resource:///org/gnome/gjs/modules/lang.js:178:13
@<command line>:10:18
```

**Comment by tchx84** [comment]
@katelk28 can you rebase on top of master? The changes from T17305 created some conflicts here.

**Pull request merged**